### DISCUSSION TITLE
How can i add custom fonts in grapesjs editor?

### DISCUSSION THREAD

**Discussion by ashok-cse** [GrapesJS/grapesjs#4563] [Q&A]
if anyone have proper guide for how to add custom font in v19.5 of grapesjs. please share the code for that

**Comment by DhanaDSP1120** [comment]
you can try this code in plugin 

> editor.StyleManager.addBuiltIn('font-family', {
>                 default: "Arial",
>                 options: [
>                     { name: 'Arial', value: 'Arial, Helvetica, sans-serif' },
>                     { name: 'Verdana', value: 'Verdana, Geneva, sans-serif' },
>                 ]
>             })
